### ISSUE TITLE
Scraping Twitter lists

### ISSUE THREAD

**Issue by nerra0pos** [JustAnotherArchivist/snscrape#849]
### Describe the feature

It seems there is no endpoint to use snscrape for Twitter lists AFAIK.

Example: https://twitter.com/i/lists/1636551680379744257

They are still publicly available (unlike search). 

Is there a plan to add these to snscrape?

### Would this fix a problem you're experiencing? If so, specify.

Yes

### Did you consider other alternatives?

Yes, search, but now defunct.

**Comment by Raiders0786** [comment]
hey @terrapop , is snscrape working for you? I have been facing difficulties!

**Comment by Raiders0786** [comment]
yes for me too, but suddenly things are not working for me and i receive same message as you posted above.

I'm referring this : https://github.com/JustAnotherArchivist/snscrape/issues/846

**Comment by Raiders0786** [comment]
yes, I am thinking to move on from snscrape to nitter.
ref: https://github.com/zedeus/nitter

Also, pls let me know if snscrape starts working for you, Thanks!

**Comment by JustAnotherArchivist** [comment]
IIRC, pagination on list pages is limited (similar to user profiles), so that's why it uses the search. If it doesn't work on the new search endpoint, I will probably implement a separate scraper that uses the list page.
